OpenRA.Mods.Common/Widgets/ButtonWidget.cs

```diff
@@ -229,9 +233,16 @@ public override void Draw()
 			var colordisabled = GetColorDisabled();
 			var bgDark = GetContrastColorDark();
 			var bgLight = GetContrastColorLight();
-			var s = font.Measure(text);
+			var textSize = font.Measure(text);
 			var stateOffset = Depressed ? new int2(VisualHeight, VisualHeight) : new int2(0, 0);
-			var position = new int2(rb.X + (UsableWidth - s.X) / 2, rb.Y - BaseLine + (Bounds.Height - s.Y) / 2);
+
+			var position = new int2(rb.X + LeftMargin, rb.Y - BaseLine + (Bounds.Height - textSize.Y) / 2);
```

Comment:
Style nit: It would be cleaner to keep the `LeftMargin` out of this line and then have a `switch(Align)` where you add the offsets for all cases (including left align).  This improves the symmetry between the three cases, and you  then don't need to subtract off the left margin in the others.